barkend-frontend/src/components/DogCard/DogCard.jsx

```diff
@@ -2,14 +2,35 @@ import './DogCard.css';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
-function DogCard({ name, size, breed, id, photo, age }) {
+function DogCard({ name, size, breed, id, photo, age, gender, fixed, kidFriendly }) {
+  
+  const getString = (category, value) => {
+    if (category === 'fixed') {
+      return (value ? "Fixed" : "Not Fixed")
+    }
+    if (category === 'kidFriendly') {
+        if (value === true) {
+          return 'Kid Friendly'
+        }
+        if (value === null) {
+          return 'Kid Status Unknown'
+        }
+        if (value === false) {
+          return 'Not Kid Friendly'
+        }
+    }
+  }
+  
+  const fixedStatus = getString('fixed', fixed);
+  const kidFriendlyStatus = getString('kidFriendly', kidFriendly);
+  
   return (
     <Link className='card-link' to={`/dog-details/${id}`}>
       <div className='dog-card'>
         <img src={photo} alt={`${name} the ${breed}`}></img>
         <h2>{name}</h2>
         <h3>{breed}</h3>
-        <h4>{age} · {size}</h4>
+        <h4>{age} · {size} · {gender} · {fixedStatus} · {kidFriendlyStatus}</h4>
```

Comment:
How do we feel about maybe NOT displaying every tag on the card itself? It feels a bit cluttered. Open to suggestions.